src/Bluejay.asm

```diff
@@ -1068,9 +1068,6 @@ ENDIF
     ljmp motor_start                    ; Go back and try starting motors again
 
 exit_run_mode_is_stall:
-    ; Enable all interrupts (disabled above, in exit_run_mode)
-    setb IE_EA
```

Comment:
Can the tidyup of interrupt handling be included in the comment message too please?

Reply:
In principle that code has been removed. So you mean to the commit message?

Reply:
Yes my apologies

Reply:
Done
![imagen](https://github.com/bird-sanctuary/bluejay/assets/2889851/bb6b109e-0331-440f-81bf-9ba001ef68e1)

Reply:
Squashing commits not be a bad idea for the future, I think @stylesuxx can turn this on on the repo side